src/common/models/cluster/cluster.ts

```diff
@@ -15,19 +15,16 @@
  * limitations under the License.
  */
 
-import { BaseImmutable, Property } from "immutable-class";
+import { BackCompat, BaseImmutable, Property } from "immutable-class";
 import { External } from "plywood";
-import { verifyUrlSafeName } from "../../utils/general/general";
+import { isTruthy, verifyUrlSafeName } from "../../utils/general/general";
 
-export type SupportedType = "druid";
 export type SourceListScan = "disable" | "auto";
-export type Protocol = "plain" | "tls-loose" | "tls";
 
 export interface ClusterValue {
   name: string;
-  type: SupportedType;
```

Comment:
Should we document that type configuration property is deprecated/ignored?

Reply:
Suport was really iffy here. We accepted only "druid" value before. Right now we silently ignore it. I would leave that out of documentation.